Jenkinsfile

```diff
@@ -340,12 +408,61 @@ pipeline {
                 }
             }    
         }
+
     */
+
     }
+
 post { 
```

Comment:
If I'm not wrong we expected to have Notifications only when we deploy/rollback or do anything else with Stage deployment, that's why I'm not sure about keeping all these notifications here in 'post'
Please verify if this meets original ticket requirements